mpas_analysis/ocean/streamfunction_moc.py

```diff
@@ -17,8 +16,9 @@
 from ..shared.timekeeping.utility import get_simulation_start_time, \
```

Comment:
Ah, I see now.  It was hidden above.

We need `open_multifile_dataset` still because it is called in `_compute_moc_time_series_postprocess` for the time series work.